scripts/A7.R

```diff
@@ -76,6 +76,15 @@ ds$data %>%
   ) -> ds$data
 
 
+# widen data --------------------------------------------------------------
+
+ds$data %>%
+  tidyr::pivot_wider(
+    names_from = c("beobachtungseinheit", "rechtsform", "geschlecht", "altersklasse"),
```

Comment:
I think this is a bit too extreme: Think how you would ask questions here: 
The column headers start to seem very unnatural like this. The `beobachtungseinheit` must for sure be wide. Every thing else I wold keep as it is.

I am not so happy with the beobachtungseinheit: after making it wide, you could map:
`Anzahl Personen am 31.12`  -> `anzahl_personen`, 'Anzahl Vollzeitäquivalente' will be mapped just fine.